### PR TITLE
Tree proximity script as of 9/4

### PR DESCRIPTION
**Pull request by stephpenn1**

### CODE REVIEW

File: R_scripts/licorstat.R [hunk 1]
```diff
@@ -28,7 +29,7 @@ dat <- filter(dat, Group == "Control")
 cat("Fitting full model...\n")
 library(nlme)
 mod_full <- nlme::lme(log(Flux) ~ T5 + T20 + SMoisture + I(SMoisture ^ 2), 
-                      random = ~ 1 | Dest_Salinity,
+                      random = ~ 1 | BAdat$BA_sqm,
```
Comment (bpbond) [inline R_scripts/licorstat.R]: I think we want this to be a fixed, not random, effect.

File: R_scripts/tree_proximity.R [hunk 2]
```diff
@@ -30,11 +32,21 @@ cat("Joining datasets...\n")
 collar_to_tree_prox <- select(proxDat, -Date) %>% 
   left_join(treeDat, by = c("Site", "Plot", "Tag"), na_matches = "never") %>%
   mutate(BA_sqm = (DBH_cm / 100 / 2) ^ 2 * pi)  # from DBH (cm) to area (m2)
+write_csv(collar_to_tree_prox, "../inventory_data/collar_to_tree_prox.csv")
+
+# Replace DBH with recorded value for non-tagged trees
+for (i in nrow(collar_to_tree_prox)) {
```
Comment (bpbond) [inline R_scripts/tree_proximity.R]: `for(i in seq_len(nrow(collar_to_tree_prox)) {`? The code above will only execute once.

Comment (bpbond) [reply R_scripts/tree_proximity.R]: You actually don't even need a loop at all:
```R
no_tags <- is.na(collar_to_tree_prox$Tag)
collar_to_tree_prox$DBH_cm[no_tags] <- collar_to_tree_prox$No_tag_DBH[no_tags]
collar_to_tree_prox$Species[no_tags] <- collar_to_tree_prox$No_tag_species[no_tags]
```

File: R_scripts/tree_proximity.R [hunk 3]
```diff
@@ -51,9 +63,14 @@ print(BA_cumdist)
...
+  facet_wrap(~ Collar) +
   ggtitle("Distribution of basal area")
 print(BA_dist)
 
-#%>%
-#  left_join(licorDat, by = "Collar")
+# Calculate cumulative basal area at each distance 
+BA_dat <- list()
+for (i in 1:10) {
+  m <- collar_to_tree_prox[which(collar_to_tree_prox$Distance_m <= 5),]
```
Comment (bpbond) [inline R_scripts/tree_proximity.R]: Shouldn't this be `<= i`?

Comment (stephpenn1) [reply R_scripts/tree_proximity.R]: Yes, I think I was testing distances out

File: R_scripts/licorstat.R (hunk 1 above)
Comment (stephpenn1) [reply R_scripts/licorstat.R]: Yes this isn't the most recent version, `master` is most recent for licorstat.R. Things are a bit messy between all three branches currently. So i wanted to merge them and start again fresh

Comment (bpbond) [reply R_scripts/licorstat.R]: You probably want to merge `master` into this first. We want to clean up any conflicts, etc., before merging _into_ master.

**Pull request merged**